lib/core/src/lib/form/components/widgets/core/form-field.model.spec.ts

```diff
@@ -881,4 +882,40 @@ describe('FormFieldModel', () => {
         });
 
     });
+
+    it('should validate readOnly field if it is validatable', () => {
+        const form = new FormModel();
+        const type = FormFieldTypes.DISPLAY_EXTERNAL_PROPERTY;
+        const field = new FormFieldModel(form, {
+            id: 'mockDisplayExternalPropertyFieldId',
+            type,
```

Comment:
I think it is better to be explicit and use FormFieldTypes.DISPLAY_EXTERNAL_PROPERTY in both places without the tmp variable